tests/WorkOS/AuditLogsTest.php

```diff
@@ -0,0 +1,193 @@
+                "Type" => "user",
+                "Name" => "User",
+            ],
+            "targets" =>
+            [
+                    "id" => "team_123",
+                    "type" => "team",
+                    "name" => "team",
+            ]];
+        $params = [
+            "organization_id" => $organizationId,
+            "event" => $auditLogEvent
+        ];
+
+        $headers = [
+            "idempotency_key" => $idempotencyKey
+        ];
+
+        $result = $this->createEventResponseFixture();
+
+        $this->mockRequest(
+            Client::METHOD_POST,
+            $path,
+            $headers,
+            $params,
+            true,
+            $result
+        );
+
+        $eventStatus = $this->al->createEvent($organizationId, $auditLogEvent);
+        $eventFixture = $this->createEventFixture();
+
+        $this->assertSame($eventFixture, $eventStatus->toArray());
+    }
+
+    public function testCreateExport()
+    {
+        $path = "audit_logs/exports";
+
+        $organizationId = "org_123";
+        $rangeStart = "2022-08-18T18:07:10.822Z";
+        $rangeEnd = "2022-08-18T18:07:10.822Z";
+        $targets = [
+            "id" => "team_123",
+            "type" => "team",
+            "name" => "team",
+        ];
+        $actions = ["document.updated"];
+        $actors = ["user_123"];
+        $params = [
+            "actions" => $actions,
+            "actors" => $actors,
+            "organization_id" => $organizationId,
+            "range_end" => $rangeEnd,
+            "range_start" => $rangeStart,
+            "targets" => $targets
+        ];
+
+        $result = $this->createExportResponseFixture();
+
+        $this->mockRequest(
+            Client::METHOD_POST,
+            $path,
+            null,
+            $params,
+            true,
+            $result
+        );
+
+        $auditLogExport = $this->al->createExport($organizationId, $rangeStart, $rangeEnd, $actions, $actors, $targets);
+        $exportFixture = $this->createExportFixture();
+
+        $this->assertSame($exportFixture, $auditLogExport->toArray());
+    }
+
+    public function testGetExport()
+    {
+        $auditLogExportId = "123";
+
+        $path = "audit_logs/exports/${auditLogExportId}";
+
+        $result = $this->getExportResponseFixture();
+
+        $this->mockRequest(
+            Client::METHOD_GET,
+            $path,
+            null,
+            null,
+            true,
+            $result
+        );
+
+        $auditLogGetExport = $this->al->getExport($auditLogExportId);
+        $getExportFixture = $this->getExportFixture();
+
+        $this->assertSame($getExportFixture, $auditLogGetExport->toArray());
+    }
+    // Fixtures
+
+    private function createEventFixture()
+    {
+        return [
+        "success" => true
+    ];
+    }
+
+    private function createEventResponseFixture()
+    {
+        return json_encode([
+        "success" => true
+    ]);
```

Comment:
nit: Could we fix the indentation across this file?

```suggestion
        return json_encode([
            "success" => true
        ]);
```

Reply:
Formatted these!